chainer/computational_graph.py

```diff
@@ -85,12 +85,16 @@ class ComputationalGraph(object):
 
     """
 
-    def __init__(self, nodes, edges, variable_style=_var_style,
-                 function_style=_func_style, rankdir='TB',
+    def __init__(self, nodes, edges, variable_style=None,
```

Comment:
I think we need to add documentations about default values of these two parameters.
Otherwise users cannot tell what can be specified (currently it can be guessed from the default value in the documentation).
https://docs.chainer.org/en/stable/reference/generated/chainer.computational_graph.ComputationalGraph.html

In addition, this breaks backward compatibility of this behavior.
https://github.com/chainer/chainer/pull/4419/files#diff-54541860b0345091e1a50777c35c730bR77